src/Workleap.Extensions.MediatR.Analyzers.Tests/SemanticDesignAnalyzerTests.cs

```diff
@@ -145,4 +145,35 @@ internal class MyNotificationHandler : INotificationHandler<MyNotification>
 }";
         await this.WithSourceCode(source).RunAsync();
     }
+
+    [Fact]
+    public async Task Notification_Handlers_Cant_Send_Query_Or_Command()
```

Comment:
The requirement is "I’d like to be allowed to write MediatR notification handlers that call other existing MediatR command and queries. "

This seems to do the opposite.

Reply:
Yeah it's not recommended to send command of queries from NotificationHandlers, but  since some teams are doing this willingly at least they can silence only GMDTR14 and keep GMDTR09 instead of silencing everything at once.

Reply:
We sorted out this misunderstanding with Eric, this behaviour is now allowed as suggested by @Zav, will rename this PR accordingly.